docs/create/import-vroid-studio.md

```diff
@@ -1,27 +1,34 @@
 ---
-id: import-vroid-studio
-title: Import From VRoid Studio
+id: import-vroid
+title: Import From VRoid
 ---
 
-This guide explains how you can export your avatar from <a href="https://vroid.com/en/studio/" target="_blank" rel="noopener noreferrer">VRoid Studio</a> to bring it into Webaverse.
+Webaverse uses [VRM](https://vrm.dev/en/), a 3D avatar file format based on the royalty free glTF2.0 standard. VRM avatars have features such as blendshapes for emotes, facial animations, physics, and are widely supported across dozens of apps and VR platforms.
```

Comment:
Maybe worth mentioning them?

Reply:
added some:

- [Cluster](https://cluster.mu/) - PCVR, Windows, macOS, iOS, Android
- [Craftopia](https://store.steampowered.com/app/1307550/Craftopia/) - Steam
- [VRAST!](https://rc-cobalt.booth.pm/items/1756359) - Windows
- [VseeFace](https://www.vseeface.icu/) - Windows
- and more: https://vrm.dev/en/vrm_applications/